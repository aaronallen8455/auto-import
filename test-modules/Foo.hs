module Foo where

class Foo where
  f1 :: ()
  b2 :: ()

type Bar = ()

baz :: ()
baz = ()
