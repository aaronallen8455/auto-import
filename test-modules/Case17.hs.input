module Case17 where

import Foo (Bar, Foo, baz)

main :: IO ()
main = f1 b2 Bar baz
