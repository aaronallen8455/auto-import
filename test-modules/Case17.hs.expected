module Case17 where

import Foo (Bar, Foo(f1, b2), baz)

main :: IO ()
main = f1 b2 Bar baz
